test/cppsuite/tests/run.cxx

```diff
@@ -65,28 +65,54 @@ value_missing_error(const std::string &str)
     test_harness::debug_print("Value missing for option " + str, DEBUG_ERROR);
 }
 
+void
+print_help()
```

Comment:
Should we call print_help from `value_missing_errror`?

Reply:
Sure